### PR TITLE
Fix:  Improve responsiveness of page #880

### LINKED ISSUE #880: [style]: Improve the responsiveness of page (text and  images)  in `/docs`
### What's the style idea?

Just a small issue in the style I have figured out here when someone tries to read the documentation by clicking the button ```Documentation```  or  ``` how to add  own projects inside the project hub```  

The text and images were partially hidden so we need to update the styles. 

Steps to reproduce the issue 
1. Inspect the website using ```ctrl + shift +i ```
2. Navigate to the documentation button inside the website
3. Choose the device toggle option in the top left of the developer options 
4. Select any small devices 

How we can fix this issue?  

Well we have two options either we would update the styles using CSS or we can replace the long images with the phone-size images using media query 

I would be happy  to resolve this issue  respected mentor please assign me @priyankarpal @hariketsheth @swapnilgarg7
Thanks 

### Add screenshots

The issue on Small sized devices like  iPhone SE  (375 X 667)  and Even in iPhone XR (414 X 896) and other devices 

image

image

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by Pursottam6003**
## Related Issue

From the issue #880  we have the information that its not  responsive on small devices  hence I have just fixed this issue using media screen 

## Description

Now for small devices its works perfectively 

## Screenshots
image

Event smaller devices 
image

### CODE REVIEW

File: src/CSS/index.css [hunk 1]
```diff
@@ -88,6 +99,25 @@ body {
...
+
+@media screen and (max-width: 390px) {
+  .block {
+    font-size: 11px;
+  }
+}
+
+@media screen and (max-width: 330px) {
+  .block {
+    font-size: 9px;
```
Comment (priyankarpal) [inline src/CSS/index.css:121]: could you add using Tailwind CSS? we are using Tailwind CSS

**Review by priyankarpal — CHANGES_REQUESTED** [review]

File: src/CSS/index.css (hunk 1 above)
Comment (Pursottam6003) [reply src/CSS/index.css]: > could you add using Tailwind CSS? we are using Tailwind CSS

Yeah sure here you go :)

Comment (Pursottam6003) [reply src/CSS/index.css]: Yes I have made changes

**Review by priyankarpal — APPROVED** [review]
Looks Good 🎉

**Pull request merged**